src/cli/configure/upgrade/index.test.ts

```diff
@@ -0,0 +1,138 @@
+import type { Dirent } from 'fs';
+import { readdir, writeFile } from 'fs/promises';
```

Comment:
Should we stick to `fs-extra`?

I initially thought of dropping it from skuba, but its use of `graceful-fs` under the hood may be a worthwhile improvement over the standard module: https://github.com/isaacs/node-graceful-fs#graceful-fs